src/connection/UndiciConnection.ts

```diff
@@ -109,7 +113,9 @@ export default class Connection extends BaseConnection {
     this.pool = new Pool(this.url.toString(), undiciOptions)
   }
 
-  async request (params: ConnectionRequestParams, options: ConnectionRequestOptions): Promise<ConnectionRequestResponse> {
+  async request (params: ConnectionRequestParams, options: ConnectionRequestOptions): Promise<ConnectionRequestResponse>
+  async request (params: ConnectionRequestParams, options: ConnectionRequestOptionsAsStream): Promise<ConnectionRequestResponseAsStream>
```

Comment:
Doesn't it mean that we need to re-generate all the API type definitions?

Reply:
Nope, the `connection.request` is only used by the transport, and its signature hasn't changed.
In the client you might need to use directly `client.transport.request<Readable>`, as the surface API assumes you are using JSON.